src/main/scala/scalafix/internal/sbt/ScalafixInterface.scala

```diff
@@ -31,6 +32,16 @@ object Arg {
         customDependencies.map(_.asCoursierCoordinates).asJava,
         repositories.asJava
       )
+
+    def mutableFiles: Seq[File] =
+      customURIs
+        .map(uri => java.nio.file.Paths.get(uri).toFile)
+        .flatMap {
+          case classDirectory if classDirectory.isDirectory =>
+            classDirectory.**(RegularFileFilter).get
```

Comment:
not directly related to this PR: this was changed from AllPassFilter to RegularFileFilter with the move, in order to skip directories